apps/wallet/src/ui/app/components/transactions-card/TxnIcon.tsx

```diff
@@ -29,30 +26,14 @@ const icons = {
         <WalletActionStake24 className="text-gradient-blue-start text-heading2 bg-transparent" />
     ),
     Unstaked: (
-        <Icon
-            icon={SuiIcons.Tokens}
-            className="text-gradient-blue-start text-heading3"
-        />
-    ),
-    Rewards: (
-        <Icon
-            icon={SuiIcons.SuiLogoIcon}
-            className="text-gradient-blue-start text-body"
-        />
+        <Unstaked className="text-gradient-blue-start text-heading6 bg-transparent" />
     ),
-    Swapped: <Swap16 className="text-gradient-blue-start text-heading6" />,
     Failed: <Info16 className="text-issue-dark text-heading6" />,
 };
 
 interface TxnItemIconProps {
     txnFailed?: boolean;
-    variant:
-        | 'Rewards'
-        | 'Staked'
-        | 'Unstaked'
-        | 'Swapped'
-        | 'Send'
-        | 'Received';
+    variant: 'Staked' | 'Unstaked' | 'Sent' | 'Received';
```

Comment:
_probably_ would either make this an enum or change how you derive it in the transaction page